monitoring/README.md

```diff
@@ -92,47 +175,124 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+- `NS: wds1-system` and `KSController: ks-transport-controller`
+
+Likewise, to monitor the KubeStellar controller in a ITS space and WEC (e.g., its1): `NS: its1-system` and `KSController: status-addon-controller` & `NS: open-cluster-management-agent-addon` and `KSController: status-agent-controller`.
+
+Note: 
+- Use the following commands to obtain the Ip address of the KubeStellar controllers in the WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces and WEC (e.g.,`cluster1`):
+
+  ```bash
+  kubectl --context kind-kubeflex -n wds1-system get Endpoints | egrep ".*controller"
+  ```
+  ```bash
+  kubectl --context kind-kubeflex -n its1-system get Endpoints | egrep ".*controller"
+  ```
+  ```bash
+  kubectl --context cluster1 -n open-cluster-management-agent-addon get Endpoints
+  ```
+
+Here are the default port number for the KubeStellar Controllers metrics endpoint: 
+
+| KS Controllers            | Default Port #|
+| ------------------------- | ------------- |
+| KS controller-manager     |     8443      |
+| transport-controller      |     8090      |
+| status-addon-controller   |     9280      |
+| status-agent-controller   |     8080      |
 
-Likewise, to monitor the KubeStellar controller in a ITS space (e.g., its1): `SpaceNS: its1-system` and `KSController: status-addon-controller`
 
 
 #### 6. View Pyroscope profile graphs for KubeStellar controllers in Grafana: 
 
 a) Connect to Pyroscope dashboard: 
-    i) Click `Connections -> Data Sources` in the primary menu
-    ii) Click `Explore` in the Pyroscope datasource selection
+   i) Click `Connections -> Data Sources` in the primary menu
+  ii) Click `Explore` in the Pyroscope datasource selection
 
-b) Query profiles for the KubeSterllar controllers: for each controller, input a profile query (see sample below) and then click the button `Run query` (on the top right).
+b) Query profiles for the Kubestellar controllers in a hosting cluster: for each controller, input a profile query (see sample below) and then click the button `Run query` (on the top right).
 - kubestellar-controller-manager: query: `{service_name="wds1-system/manager"}`
 - transport-controller: query: `{service_name="wds1-system/transport-controller"}`
-- addon-status-controller: query: `{service_name="its1-system/status-controller"}` (warning: graph does not display - WIP)
+- status-addon-controller: query: `{service_name="its1-system/status-controller"}`
 
     <img src="images/ks-controllers-pyroscope.png" width="60%" height="80%" title="ks-controllers-pyroscope">
 
+c) Query profiles for the Kubestellar controller in a WEC cluster: connect to the UI of the Pyroscope application running in the target WEC, since there is not a single pane of glass to visualize Pyroscope data across all clusters in our monitoring architecture:
+
+  i) Port-forward Pyroscope to localhost, by using the kubectl command:
+
+  ```bash
+  kubectl --context cluster1 -n ks-monitoring port-forward svc/pyroscope 9090:4040
+  ```
```

Comment:
FYI, I think that it would be helpful at this point to explicitly tell the user to direct a browser to http://localhost:9090/ (readers might not remember whether the 9090 or the 4040 is the port on the local side).